x-pack/plugins/discover_log_explorer/public/deep_links.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AppDeepLink, AppNavLinkStatus, DEFAULT_APP_CATEGORIES } from '@kbn/core/public';
+import { i18n } from '@kbn/i18n';
+import { LOG_EXPLORER_PROFILE_ID } from '../common/constants';
+
+export const getLogExplorerDeepLink = ({ isVisible }: { isVisible: boolean }): AppDeepLink => ({
+  id: LOG_EXPLORER_PROFILE_ID,
+  title: i18n.translate('xpack.discoverLogExplorer.deepLink', {
+    defaultMessage: 'Logs Explorer',
+  }),
+  path: `#/p/${LOG_EXPLORER_PROFILE_ID}`,
```

Comment:
I agree that this would be preferable, but the async nature of the locator function would have to bubble all the way up to the `setup()` call. That would be pretty awkward, so I'd rather leave it as it is right now if you'd be fine with that.

Reply:
That makes sense and not a problem by me 🙂 Figured I'd point it out just in case.